src/neon/stage1.rs

```diff
@@ -1,18 +1,19 @@
 #![allow(dead_code)]
 
-use crate::neon::intrinsics::*;
 use crate::neon::utf8check::*;
 use crate::*;
+use std::arch::aarch64::*;
+use simd_lite::aarch64::*;
 
 use std::mem;
 
 // NEON-SPECIFIC
 
 macro_rules! bit_mask {
     () => {
-        uint8x16_t::new(
-            0x01, 0x02, 0x4, 0x8, 0x10, 0x20, 0x40, 0x80,
-            0x01, 0x02, 0x4, 0x8, 0x10, 0x20, 0x40, 0x80
+       mem::transmute::<[u8;16],uint8x16_t>(
```

Comment:
this leaves a sour taste, I'm thinking a developer shouldn't have to use transmute for simple day-to-day tasks

Reply:
you're right we should use a load instruction here! :)

Reply:
I'm up for whatever, just be aware of my plan to adopt all the types so they can be made more friendly  :)